src/constants/skills.js

```diff
@@ -201,6 +210,16 @@ const categorizedSkills = {
     ],
   },
 
+  fullstack_dev: {
+    title: "Fullstack Development",
+    skills: [
+      "MERN_stack",
+      "MEAN_stack",
+      "LAMP_stack",
+      "JAM_stack",
```

Comment:
Change these to lowercase as all the other skills are following the same style.

Reply:
all the changes are done.